### ISSUE TITLE
Running `wsl code <path>` in terminal does not return (works as with `-w` option)

### ISSUE THREAD

**Issue by kanlukasz** [microsoft/vscode#193337]
### Description:
Something happened with running VSCode in WSL via the `wsl code /var/www/app` command.
Until now, after executing this command, VSCode would open in the WSL environment according to the specified path and then release the terminal. That is, it was possible to give another command in the terminal or close it.
Currently, this command behaves as if I had added the `-w` flag.
That is, the terminal does not release this command. It does so only after I close VSCode

### ScreenGIF:
![wsl](https://github.com/microsoft/vscode/assets/30685349/f6b61b64-0e74-4ba6-b3bb-8dbae12acc14)

### VS Code Version:
```
Version: 1.82.2
Commit: abd2f3db4bdb28f9e95536dfa84d8479f1eb312d
Date: 2023-09-14T05:55:25.390Z
Browser: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Code/1.82.2 Chrome/114.0.5735.289 Electron/25.8.1 Safari/537.36
```

### OS Version: 
```
OS Name:                   Microsoft Windows 11 Pro
OS Version:                10.0.22621 N/A Build 22621
```

### WSL Version:
```
WSL version: 1.2.5.0
Kernel version: 5.15.90.1
WSLg version: 1.0.51
MSRDC version: 1.2.3770
Direct3D version: 1.608.2-61064218
DXCore version: 10.0.25131.1002-220531-1700.rs-onecore-base2-hyp
Windows version: 10.0.22621.2283
```

### WSL Extension version:
```
v0.81.3
```

### Potential related issues and duplicates:
- https://github.com/microsoft/vscode/issues/193332 
- https://github.com/microsoft/vscode/issues/193321
- https://github.com/microsoft/vscode/issues/193302
- https://github.com/microsoft/vscode/issues/62698 
- https://github.com/microsoft/vscode/issues/148582

**Comment by Temepest74** [comment]
+1

It happens even when you go the the WSL distro console and type ```code /app```

**Comment by KnisterPeter** [comment]
Same as https://github.com/microsoft/vscode/issues/188426?

**Comment by lamuertepeluda** [comment]
> Same as #188426?

and same as #193328

**Comment by kanlukasz** [comment]
> and same as #193328

I don't see any connection between my issue and this.

**Comment by lamuertepeluda** [comment]
Sorry guys I meant #193332 I was miscarried by gh autosuggest while typing

**Comment by samuelshenrique** [comment]
+1 here

**Comment by ChristopherGerdes** [comment]
+1 on this issue with win11, wsl2 on a brand new machine.

**Comment by aeschli** [comment]
duplicate of #193453
